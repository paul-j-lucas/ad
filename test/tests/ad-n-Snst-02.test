ad | -c always -n -Snst | Waldo.txt | | 0
