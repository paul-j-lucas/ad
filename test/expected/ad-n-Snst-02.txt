[32m[K0000000000000000[m[K[36m[K:[m[K [41;1m[K5761[m[K [41;1m[K6C64[m[K [41;1m[K6F2E[m[K [41;1m[K2E2E[m[K  [41;1m[K2E2E[m[K [41;1m[K2E2E[m[K [41;1m[K2E2E[m[K [41;1m[K2E0A[m[K  [41;1m[KWaldo...........[m[K
[32m[K0000000000000010[m[K[36m[K:[m[K [41;1m[K2057[m[K [41;1m[K616C[m[K [41;1m[K646F[m[K [41;1m[K2E2E[m[K  [41;1m[K2E2E[m[K [41;1m[K2E2E[m[K [41;1m[K2E2E[m[K [41;1m[K2E0A[m[K  [41;1m[K Waldo..........[m[K
[32m[K0000000000000020[m[K[36m[K:[m[K [41;1m[K2020[m[K [41;1m[K5761[m[K [41;1m[K6C64[m[K [41;1m[K6F2E[m[K  [41;1m[K2E2E[m[K [41;1m[K2E2E[m[K [41;1m[K2E2E[m[K [41;1m[K2E0A[m[K  [41;1m[K  Waldo.........[m[K
[32m[K0000000000000030[m[K[36m[K:[m[K [41;1m[K2020[m[K [41;1m[K2057[m[K [41;1m[K616C[m[K [41;1m[K646F[m[K  [41;1m[K2E2E[m[K [41;1m[K2E2E[m[K [41;1m[K2E2E[m[K [41;1m[K2E0A[m[K  [41;1m[K   Waldo........[m[K
[32m[K0000000000000040[m[K[36m[K:[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K [41;1m[K5761[m[K [41;1m[K6C64[m[K  [41;1m[K6F2E[m[K [41;1m[K2E2E[m[K [41;1m[K2E2E[m[K [41;1m[K2E0A[m[K  [41;1m[K    Waldo.......[m[K
[32m[K0000000000000050[m[K[36m[K:[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K [41;1m[K2057[m[K [41;1m[K616C[m[K  [41;1m[K646F[m[K [41;1m[K2E2E[m[K [41;1m[K2E2E[m[K [41;1m[K2E0A[m[K  [41;1m[K     Waldo......[m[K
[32m[K0000000000000060[m[K[36m[K:[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K [41;1m[K5761[m[K  [41;1m[K6C64[m[K [41;1m[K6F2E[m[K [41;1m[K2E2E[m[K [41;1m[K2E0A[m[K  [41;1m[K      Waldo.....[m[K
[32m[K0000000000000070[m[K[36m[K:[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K [41;1m[K2057[m[K  [41;1m[K616C[m[K [41;1m[K646F[m[K [41;1m[K2E2E[m[K [41;1m[K2E0A[m[K  [41;1m[K       Waldo....[m[K
[32m[K0000000000000080[m[K[36m[K:[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K  [41;1m[K5761[m[K [41;1m[K6C64[m[K [41;1m[K6F2E[m[K [41;1m[K2E0A[m[K  [41;1m[K        Waldo...[m[K
[32m[K0000000000000090[m[K[36m[K:[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K  [41;1m[K2057[m[K [41;1m[K616C[m[K [41;1m[K646F[m[K [41;1m[K2E0A[m[K  [41;1m[K         Waldo..[m[K
[32m[K00000000000000A0[m[K[36m[K:[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K  [41;1m[K2020[m[K [41;1m[K5761[m[K [41;1m[K6C64[m[K [41;1m[K6F0A[m[K  [41;1m[K          Waldo.[m[K
[32m[K00000000000000B0[m[K[36m[K:[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K  [41;1m[K2020[m[K [41;1m[K2057[m[K [41;1m[K616C[m[K [41;1m[K646F[m[K  [41;1m[K           Waldo[m[K
[32m[K00000000000000C0[m[K[36m[K:[m[K [41;1m[K0A20[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K  [41;1m[K2020[m[K [41;1m[K2020[m[K [41;1m[K5761[m[K [41;1m[K6C64[m[K  [41;1m[K.           Wald[m[K
[32m[K00000000000000D0[m[K[36m[K:[m[K [41;1m[K6F0A[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K  [41;1m[K2020[m[K [41;1m[K2020[m[K [41;1m[K2057[m[K [41;1m[K616C[m[K  [41;1m[Ko.           Wal[m[K
[32m[K00000000000000E0[m[K[36m[K:[m[K [41;1m[K646F[m[K [41;1m[K0A20[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K  [41;1m[K2020[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K [41;1m[K5761[m[K  [41;1m[Kdo.           Wa[m[K
[32m[K00000000000000F0[m[K[36m[K:[m[K [41;1m[K6C64[m[K [41;1m[K6F0A[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K  [41;1m[K2020[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K [41;1m[K2057[m[K  [41;1m[Kldo.           W[m[K
[32m[K0000000000000100[m[K[36m[K:[m[K [41;1m[K616C[m[K [41;1m[K646F[m[K [41;1m[K0A20[m[K [41;1m[K2020[m[K  [41;1m[K2020[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K [41;1m[K2020[m[K  [41;1m[Kaldo.           [m[K
[32m[K0000000000000110[m[K[36m[K:[m[K [41;1m[K5761[m[K [41;1m[K6C64[m[K [41;1m[K6F0A[m[K                            [41;1m[KWaldo.[m[K
